lib/npmrc.js

```diff
@@ -2,23 +2,42 @@ var asp = require('rsvp').denodeify;
 var fs = require('graceful-fs');
 var path = require('path');
 var utils = require('./utils');
+var which = require('which');
 
-function getFilepath() {
+function getFilepaths() {
   var home = process.env.HOME || process.env.USERPROFILE || process.env.HOMEPATH;
```

Comment:
This should be `isWindows ? process.env.USERPROFILE : process.env.HOME` as in https://github.com/npm/osenv/blob/master/osenv.js#L53.